src/framework/json.hpp

```diff
@@ -22,9 +22,20 @@
 #include <map>
 #include <vector>
 
+#include <pybind11/pybind11.h>
```

Comment:
Can we compartmentalize this so that the standalone simulator doesn't require Pybind to build?

Reply:
yes - didn't realize this would have the effect of introducing pointless dependencies to the standalone but now I do - I can move the pybind stuff to its own header file and only include it from the wrappers

Reply:
addressed in https://github.com/Qiskit/qiskit-aer/pull/399/commits/7c0784fb72e7cbff2cc9b1d51ab10a2f991bb1ff